src/store/actions.ts

```diff
@@ -0,0 +1,14 @@
+import { ActionTree } from 'vuex';
+
+import { VQBState } from './state';
+
+const actions: ActionTree<VQBState, any> = {
```

Comment:
genuine question: do we need an action here? I though actions where primarily used for asynchronous mutations.

Reply:
> Actions _can_ contain arbitrary asynchronous operations.
(https://vuex.vuejs.org/guide/actions.html)
Mainly yes, but they also serve the purpose to compose multiple mutations.
In my understanding, it's a way to a extract higher-order functions. The only other way to factor multiple mutations would be to extract them as functions, call these functions in their mutation definition, and calls them also int he higher-order mutation that composes them. I seems easier to me tp use vuex actions for this.

Reply:
LGTM, thanks for the explanation!